content/en/docs/concepts/traffic-management/index.md

```diff
@@ -885,12 +511,10 @@ spec:
 {{< /text >}}
 
 This gateway configuration lets HTTPS traffic from `ext-host` into the mesh on
-port 443, but doesn't specify any routing for the traffic.
-
-#### Bind a gateway to a virtual service
+port 443, but doesn’t specify any routing for the traffic.
 
 To specify routing and for the gateway to work as intended, you must also bind
-the gateway to a virtual service. You do this using the virtual service's
+the gateway to a virtual service. You do this using the virtual service’s
 `gateways` field, as shown in the following example:
 
 {{< text yaml >}}
```

Comment:
The host in the virtual service needs to match the gateway host ext-host (or ext-host.example.com), not ext-svc

Reply:
Ditto!